src/current/_includes/releases/v24.3/v24.3.12.md

```diff
@@ -0,0 +1,63 @@
+- Fixed a bug where CockroachDB would encounter an internal error when decoding the gists of plans with `CALL` statements. The bug had been present since v23.2. [#143314][#143314]
+- The reader virtual cluster now starts if the user begins a physical cluster replication (PCR) stream from a cursor via `ALTER VIRTUAL CLUSTER virtual_cluster START REPLICATION OF virtual_cluster ON pgurl_physical_cluster WITH READ VIRTUAL CLUSTER`. [#143369][#143369]
+- Fixed a crash due to "use of enum metadata before hydration" when using logical data replication (LDR) with user-defined types (UDTs). [#143376][#143376]
+- Fixed a potential deadlock that could occur during client certificate updates while metrics were being collected. This issue affected the reliability of certificate expiration reporting. [#143591][#143591]
+- Fixed a bug in `v24.1.14`, `v24.3.7`, `v24.3.8`, and `v25.1` that could cause a nil-pointer error when a column's default expression contained a volatile expression (like `nextval`) as a UDF argument. [#143636][#143636]
+- Previously, the fields `maximum memory usage` and `max sql temp disk usage` in the `EXPLAIN ANALYZE` output could be under-reported for distributed plans when memory-intensive operations were fully performed on the remote nodes. This is now fixed. The bug existed in v22.1 and later. [#143793][#143793]
+- The ALTER VIRTUAL CLUSTER SET REPLICATION READ VIRTUAL CLUSTER syntax is now supported for adding a reader virtual cluster for an existing PCR standby. [#143905][#143905]
+- Previously, whenever CockroachDB collected a statement bundle when plan-gist-based matching was used, `plan.txt` would be incomplete. This is now fixed. The bug had been present since the introduction of plan-gist-based matching in v23.1, and was partially addressed in v24.2. [#143935][#143935]
+- Fixed a bug where CockroachDB could encounter a `cannot specify timestamp older than ...` error during table statistics collection in some cases (e.g., when the cluster is overloaded). The bug was present since v19.1. [#144017][#144017]
+- Fixed a bug that could cause a stack overflow during execution of a prepared statement that invoked a PL/pgSQL routine with a loop. The bug existed in versions v23.2.22, v24.1.15, v24.3.9, v25.1.2, v25.1.3, and pre-release versions of v25.2 prior to v25.2.0-alpha.3. [#144030][#144030]
+
+<h3 id="v24-3-12-miscellaneous">Miscellaneous</h3>
+
+- Fixed a rare corruption bug that impacts `IMPORT` and materialized views. [#144688][#144688]
+- SQL queries run on the source cluster by logical data replication (LDR) and physical cluster replication (PCR) will account to internal metrics like `sql.statements.active.internal` instead of the metrics like `sql.statements.active` that are used to monitor application workload. [#145115][#145115]
```

Comment:
I'm not sure this note is entirely clear, but I could be misreading.

Reply:
it's ... weird, but i'm leaving it on the theory that supposedly it was edited by the relevant TW